examples/binary/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+## Setting up your working environment
+
+binary requires OCaml 4.09.0 or higher so you will need a corresponding opam
+switch. OCaml 4.09.0 is a good choice as it makes for a better developper
+experience thanks to improved error messages, amongst other things. You can
```

Comment:
I don't think this middle sentence is necessary, but if you think so there's a typo in `developer`.